### PR TITLE
[10.0][IMP] shopinvader locomotive change password

### PR DESCRIPTION
**Pull request by hparfr**
Should be merged after #255 

Change locomotiveCMS password from a wizard instead of directly in the form view.
screenshot_2018-12-12 demo shopinvader backend - odoo
screenshot_2018-12-12 demo shopinvader backend - odoo 1

### CODE REVIEW

File: shopinvader_locomotive/models/shopinvader_backend.py [hunk 1]
```diff
@@ -37,3 +37,29 @@ def _export_metafields_store(self, fields=None):
                 exporter = work.component(usage='record.exporter')
                 exporter.run(fields=fields)
         return True
+
+
+class ChangePasswordWizard(models.TransientModel):
```
Comment (lmignon) [inline shopinvader_locomotive/models/shopinvader_backend.py:42]: this wizard is already declared into /wizard/change_password.py

File: shopinvader_locomotive/wizard/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from . import change_password
```
Comment (lmignon) [inline shopinvader_locomotive/wizard/__init__.py:1]: directory should be `wizards`

File: shopinvader_locomotive/wizard/change_password.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Akretion (http://www.akretion.com).
+# @author Raphaël Reverdy <raphael.reverdy@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ChangePasswordWizard(models.TransientModel):
```
Comment (lmignon) [inline shopinvader_locomotive/wizard/change_password.py:9]: The name of the module should be `shopinvader....`
The name of the class should be `ShopinvaderChange...`
The name of the file should be `shopinvader_shange....py`

**Review by lmignon — CHANGES_REQUESTED** [review]

**Comment by sebastienbeau** [comment]
Not sure it worth the complexity.
In 12 most of people will use server env and for keychain user I plan to add a generic wizard.
So this will be only usable for user that do not use keychain and not server env.
Note we are going to try to have the maximun code in commun between the two version

**Comment by lmignon** [comment]
@sebastienbeau @hparfr I propose to close this PR since the way pwd are managed will change

**Comment by hparfr** [comment]
I think it's even better to never let the user define this password.
It should be generated by odoo. And better pushed to rails.

@sebastienbeau If you wan't to keep the broken UI design for v10, I'm happy with that ;-)

**Pull request closed without merging**